### PR TITLE
Remove transparent padding from the status icon image files.

### LINKED ISSUE #8335: [UI] Side bar Unit Image is no longer centered
image

The screenshot is self-explanatory.
It's not centered anymore, but rather placed to the top left.

### PR DESCRIPTION
**Pull request by Pentarctagon**
Just removing transparency from all images on the top bar and side bar causes other issues, particularly with the unit sprite and the clock icon alignment.

Fixes #8335

### CODE REVIEW

**Comment by CelticMinstrel** [comment]
Why are you reverting the previous change? I don't think there's any actual reason to remove the new IPF, regardless of whether we're going to apply it to all images.

**Comment by Pentarctagon** [comment]
The only reason it was added was to remove transparent pixel padding for the top bar/side bar images. I don't think there's really a need to keep it around otherwise?

**Comment by CelticMinstrel** [comment]
> I don't think there's really a need to keep it around otherwise?

I'm sure there are other use-cases for it though? If nothing else, it's a quick and easy way for people to update their custom status icons to not be squashed in 1.18. There are probably other use-cases too. Maybe it could be useful for composing add-on or achievement images that use large units. Maybe it's useful for something where people want an image to be left-aligned instead of centred.

In summary, even if the original use-case is gone, I don't think there's a need to remove it.

**Comment by Pentarctagon** [comment]
Alright, re-added.

**Comment by knyghtmare** [comment]
what is the new dimensions for status icons?
I see images have now 10-16 x 10-16 sizes. is it 16x16? or 14x14?

**Comment by CelticMinstrel** [comment]
That's certainly one disadvantage of this change… the icons no longer have standardized sizes…

**Comment by Pentarctagon** [comment]
The UtBS dehydration icon is 18x18. It's not listed here since it had no transparent padding to remove.
